{"pytest_version": "7.2.2", "$report_type": "SessionStart"}
{"nodeid": "tests58_32_3.py::test_matrix_of_one_integer", "location": ["tests58_32_3.py", 18, "test_matrix_of_one_integer"], "keywords": {"test_matrix_of_one_integer": 1, "__wrapped__": 1, "tests58_32_3.py": 1, "Folder_32": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00024743399990256876, "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "db885dc01b844177b5d8a58cf7193d35", "node": "<WorkerController gw0>"}
{"nodeid": "tests58_32_3.py::test_horizontal_matrix", "location": ["tests58_32_3.py", 40, "test_horizontal_matrix"], "keywords": {"test_horizontal_matrix": 1, "__wrapped__": 1, "tests58_32_3.py": 1, "Folder_32": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00024354599736398086, "$report_type": "TestReport", "item_index": 2, "worker_id": "gw2", "testrun_uid": "db885dc01b844177b5d8a58cf7193d35", "node": "<WorkerController gw2>"}
{"nodeid": "tests58_32_3.py::test_vertical_matrix", "location": ["tests58_32_3.py", 60, "test_vertical_matrix"], "keywords": {"test_vertical_matrix": 1, "__wrapped__": 1, "tests58_32_3.py": 1, "Folder_32": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00022639300004811957, "$report_type": "TestReport", "item_index": 3, "worker_id": "gw3", "testrun_uid": "db885dc01b844177b5d8a58cf7193d35", "node": "<WorkerController gw3>"}
{"nodeid": "tests58_32_3.py::test_matrix_of_having_103_integers", "location": ["tests58_32_3.py", 28, "test_matrix_of_having_103_integers"], "keywords": {"test_matrix_of_having_103_integers": 1, "__wrapped__": 1, "tests58_32_3.py": 1, "Folder_32": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.0023767970051267184, "$report_type": "TestReport", "item_index": 1, "worker_id": "gw1", "testrun_uid": "db885dc01b844177b5d8a58cf7193d35", "node": "<WorkerController gw1>"}
{"nodeid": "tests58_32_3.py::test_matrix_of_having_103_integers", "location": ["tests58_32_3.py", 28, "test_matrix_of_having_103_integers"], "keywords": {"test_matrix_of_having_103_integers": 1, "__wrapped__": 1, "tests58_32_3.py": 1, "Folder_32": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/home/shahin/.local/lib/python3.10/site-packages/numpy/core/_methods.py", "lineno": 49, "message": "numpy.AxisError: axis 2 is out of bounds for array of dimension 2"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    @ignore_warnings", "    def test_matrix_of_having_103_integers():", "        if 103 % 2 == 0:", "            input_matrix = [[random.randint(-100, 100) for _ in range(2)] for _ in range(103 // 2)]", "            input_matrix = np.asmatrix(input_matrix)", "        else:", "            input_matrix = [[random.randint(-100, 100) for _ in range(103)]]", "            input_matrix = np.asmatrix(input_matrix)", "    ", ">       assert submatrix_with_n_numbers(input_matrix) == 1"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q58/command_results_3/Folder_32/tests58_32_3.py", "lineno": 38, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    return np.sum(np.all(np.sum(np.sum(matrix, axis=1), axis=2), axis=3) == 103)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q58/command_results_3/Folder_32/generated_answer.py", "lineno": 3, "message": "in submatrix_with_n_numbers"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    ???"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "<__array_function__ internals>", "lineno": 200, "message": "in sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return _wrapreduction(a, np.add, 'sum', axis, dtype, out, keepdims=keepdims,"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "../.local/lib/python3.10/site-packages/numpy/core/fromnumeric.py", "lineno": 2324, "message": "in sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return reduction(axis=axis, out=out, **passkwargs)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "../.local/lib/python3.10/site-packages/numpy/core/fromnumeric.py", "lineno": 84, "message": "in _wrapreduction"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return N.ndarray.sum(self, axis, dtype, out, keepdims=True)._collapse(axis)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "../.local/lib/python3.10/site-packages/numpy/matrixlib/defmatrix.py", "lineno": 319, "message": "in sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def _sum(a, axis=None, dtype=None, out=None, keepdims=False,", "             initial=_NoValue, where=True):", ">       return umr_sum(a, axis, dtype, out, keepdims, initial, where)", "E       numpy.AxisError: axis 2 is out of bounds for array of dimension 2"], "reprfuncargs": {"args": [["a", "matrix([[727]])"], ["axis", "2"], ["dtype", "None"], ["out", "None"], ["keepdims", "True"], ["initial", "<no value>"], ["where", "True"]]}, "reprlocals": null, "reprfileloc": {"path": "../.local/lib/python3.10/site-packages/numpy/core/_methods.py", "lineno": 49, "message": "AxisError"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    @ignore_warnings", "    def test_matrix_of_having_103_integers():", "        if 103 % 2 == 0:", "            input_matrix = [[random.randint(-100, 100) for _ in range(2)] for _ in range(103 // 2)]", "            input_matrix = np.asmatrix(input_matrix)", "        else:", "            input_matrix = [[random.randint(-100, 100) for _ in range(103)]]", "            input_matrix = np.asmatrix(input_matrix)", "    ", ">       assert submatrix_with_n_numbers(input_matrix) == 1"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q58/command_results_3/Folder_32/tests58_32_3.py", "lineno": 38, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    return np.sum(np.all(np.sum(np.sum(matrix, axis=1), axis=2), axis=3) == 103)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q58/command_results_3/Folder_32/generated_answer.py", "lineno": 3, "message": "in submatrix_with_n_numbers"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    ???"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "<__array_function__ internals>", "lineno": 200, "message": "in sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return _wrapreduction(a, np.add, 'sum', axis, dtype, out, keepdims=keepdims,"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "../.local/lib/python3.10/site-packages/numpy/core/fromnumeric.py", "lineno": 2324, "message": "in sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return reduction(axis=axis, out=out, **passkwargs)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "../.local/lib/python3.10/site-packages/numpy/core/fromnumeric.py", "lineno": 84, "message": "in _wrapreduction"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return N.ndarray.sum(self, axis, dtype, out, keepdims=True)._collapse(axis)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "../.local/lib/python3.10/site-packages/numpy/matrixlib/defmatrix.py", "lineno": 319, "message": "in sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def _sum(a, axis=None, dtype=None, out=None, keepdims=False,", "             initial=_NoValue, where=True):", ">       return umr_sum(a, axis, dtype, out, keepdims, initial, where)", "E       numpy.AxisError: axis 2 is out of bounds for array of dimension 2"], "reprfuncargs": {"args": [["a", "matrix([[727]])"], ["axis", "2"], ["dtype", "None"], ["out", "None"], ["keepdims", "True"], ["initial", "<no value>"], ["where", "True"]]}, "reprlocals": null, "reprfileloc": {"path": "../.local/lib/python3.10/site-packages/numpy/core/_methods.py", "lineno": 49, "message": "AxisError"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/home/shahin/.local/lib/python3.10/site-packages/numpy/core/_methods.py", "lineno": 49, "message": "numpy.AxisError: axis 2 is out of bounds for array of dimension 2"}, null]]}, "when": "call", "user_properties": [], "sections": [], "duration": 0.0005839369987370446, "extra": [], "$report_type": "TestReport", "item_index": 1, "worker_id": "gw1", "testrun_uid": "db885dc01b844177b5d8a58cf7193d35", "node": "<WorkerController gw1>"}
{"exitstatus": 2, "$report_type": "SessionFinish"}
